plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/plan/CubridPlanNode.java

```diff
@@ -0,0 +1,196 @@
+        this.fullText = fullText;
+        parseObject(segments);
+        parseNode();
+    }
+
+    private void parseObject(List<String> segments) 
+    {
+        if (!segments.isEmpty()) {
+            String[] removes = segments.remove(0).split(":");
+            nodeProps.put(removes[0], removes[1].trim());
+            if (removes[0].equals("cost") || segments.isEmpty()) {
+                return;
+            }
+            String key = segments.get(0).split(":")[0];
+            if (nodeProps.containsKey(key) || removes[0].equals("subplan")) {
+                addNested(removes[1].trim(), segments);
+                parseObject(segments);
+            } else if (key.equals("class")) {
+                if (!removes[0].equals("Query plan")) {
+                    addNested(removes[1].trim(), segments);
+                }
+                parseObject(segments);
+            } else {
+                parseObject(segments);
+            }
+        }
+    }
+
+    private void addNested(String name, List<String> value) 
+    {
+        if (nested == null) {
+            nested = new ArrayList<>();
+        }
+        nested.add(new CubridPlanNode(this, name, value, fullText));
+    }
+
+    @Property(order = 0, viewable = true)
+    @Override
+    public String getNodeType() 
+    {
+        return getMethodTitle(name);
+    }
+
+    @Property(order = 1, viewable = true)
+    @Override
+    public String getNodeName() 
+    {
+        return classNode.get(nodeName);
+    }
+
+    @Property(order = 2, viewable = true)
+    public Number getCost() 
+    {
+        return cost;
+    }
+
+    @Property(order = 3, viewable = true)
+    public Number getCardinality() 
+    {
+        return row;
+    }
+
+    @Property(order = 4, length = PropertyLength.MULTILINE)
+    public String getFullText() 
+    {
+        return fullText;
+    }
+
+    @Override
+    public CubridPlanNode getParent() 
+    {
+        return parent;
+    }
+
+    @Override
+    public Collection<CubridPlanNode> getNested() 
+    {
+        return nested;
+    }
+
+    public String getMethodTitle(String method) 
+    {
+
+        switch (method) {
+            case "iscan":
+                return "Index Scan";
+            case "sscan":
+                return "Full Scan";
+            case "temp(group by)":
+                return "Group by Temp";
+            case "temp(order by)":
+                return "Order by Temp";
+            case "nl-join (inner join)":
+                return "Nested Loop - Inner Join";
+            case "nl-join (cross join)":
+                return "Nested Loop - Cross Join";
+            case "idx-join (inner join)":
+                return "Index Join - Inner Join";
+            case "m-join (inner join)":
+                return "Merged - Inner Join";
+            case "temp":
+                return "Temp";
+            case "follow":
+                return "Follow";
+            default:
+                return method;
+        }
```

Comment:
Switch expression can be used instead:

```suggestion
        return switch (method) {
            case "iscan" -> "Index Scan";
            case "sscan" -> "Full Scan";
            case "temp(group by)" -> "Group by Temp";
            case "temp(order by)" -> "Order by Temp";
            case "nl-join (inner join)" -> "Nested Loop - Inner Join";
            case "nl-join (cross join)" -> "Nested Loop - Cross Join";
            case "idx-join (inner join)" -> "Index Join - Inner Join";
            case "m-join (inner join)" -> "Merged - Inner Join";
            case "temp" -> "Temp";
            case "follow" -> "Follow";
            default -> method;
        };
```